corona-warn-app/cwa-app-android
# Fix exception when deselecting person (EXPOSUREAPP-6003)

**Steps to test**
- Open locations or persons
- Select an existing item
- Click in the notes field, so the cursor is shown
- Deselect the person or location, selected in step 2

**Expected result:**
- No exception



**Additional Remarks:**
When editing notes, the soft keyboard shows up. If another entry (e.g. duration) is edited afterward, the soft keyboard should be hidden.
